### DISCUSSION TITLE
unacceptableStatusCode(401) when trying to view posts by an unauthorized user

### DISCUSSION THREAD

**Discussion by avldokuchaev** [supabase/supabase#16285] [Questions]
The database gives data only to authorized users, and does not give it to unauthorized users. Writes an error in the application logs unacceptableStatusCode(401). I have already disabled RLS for all tables but still the same.
How can i fix it?

**Comment by GaryAustin1** [comment]
You are going to need to provide more info for someone to help you.  What are you doing when you get the error?  What is your environment? Have you changed grants on any tables?  Does authorized mean signed in user like an authenticated user?  What logs are application logs?

**Reply by avldokuchaev** [reply]
I am writing an app in swiftui. When the user is logged in, that is, he enters the application using e-mail and a password, then the data from the supabase database is displayed normally. There are no errors. And when the user is not authorized, that is, when the main page of the application is shown, where posts should also be shown, I get the error Response status code was unacceptable: 401.

**Reply by avldokuchaev** [reply]
my policies 
ALL Enable read access for all users
Applied to: public

**Comment by avldokuchaev** [comment]
My supabase into my VPS server. I checked the application in the cloud, it works there. That is, I understand that my server is not correctly configured. I looked at the Logs of the nginx server and saw such entries there. 

[07/Aug/2023:21:47:47 +0300] "GET /rest/v1/Posts?select=*&order=upPostPlusDayValueCount.desc.nullslast&limit=3 HTTP/2.0" 401 87 "-" "TicDog/1 CFNetwork/1402.0.8 Darwin/21.6.0"

If someone understands what the problem is, please tell me.
